test/api/test_op_construct.cpp

```diff
@@ -49,4 +49,11 @@ TEST_CASE(reduce_mean_with_format)
     EXPECT(rm.name() == "reduce_mean");
 }
 
+TEST_CASE(allocate_with_nested_json_str)
+{
+    auto alloc = migraphx::operation(
+        "allocate", R"({"shape":{"type":"float_type","lens":[3, 3], "strides":[3, 1]}})");
```

Comment:
I dont think this is a good test. We shouldn't rely on how the shape class is serialized, as it will change in the future. I think we can remove this.

Reply:
Removed that test.